modules/ROOT/pages/install-self-managed-network-configuration.adoc

```diff
@@ -32,9 +32,9 @@ The following table does not include ports and hostnames from Anypoint Platform
 
 The Persistent Gateway requires a Postgres-compliant database to store persistent data across Mule application replicas. Ensure that your Kubernetes cluster has access to this database and port. See xref:persistence-gateway.adoc[Persistence Gateway].
 
-== Hostname Configuration
+== Hostname Endpoints Configuration
```

Comment:
This would typically be Hostname Endpoint Configuration (Endpoint singular). 
Rule: No plural for adjectives:
Adjectives in English are always singular, so when a noun takes on an adjectival role, it follows this rule and remains singular.